src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -63,6 +63,14 @@ tests:
       command: "LockDoor"
       timedInteractionTimeoutMs: 1000
 
+    - label: "Wait for lock action to complete"
```

Comment:
These are certification tests. We can't just change them like this without updating the test plan, right?

That said, they are broken as written, looks like, and would fail with real locks.  What I would suggest is disabling them in CI for now and filing a test plan issue to get them fixed....  Or just fixing the test plan.

Reply:
Also, if we switch to 0 delay it's possible that the tests will pass in CI for now... and then we just need to get the test plan fixed in general.